Automattic/wp-calypso
# Add new promo nudge and banner for upcoming May sale

#### Changes proposed in this Pull Request

*  Add new promo nudge and banner for upcoming May sale. The sale will be for wpcom and Jetpack.

#### Testing instructions

* With this branch, edit the lines (60 and 74) that define the `startsAt` for the two new entries so the date is before today.
  * eg. Change to: `startsAt: new Date( 2019, 0, 25, 0, 0, 0 ),`
* Sandbox `public-api.wordpress.com`.
* Apply this patch on the backend: D28493-code
* From the backend, set two attributes on your user:
```
update_user_attribute( YOUR_USER_ID, 'nudge_tag_sale_may_2019', 'send' )
update_user_attribute( YOUR_USER_ID, 'nudge_tag_sale_may_2019_jp', 'send' )
```
* For a wpcom site on free, Blogger, Personal, or Premium, go to `http://calypso.localhost:3000/stats/day/[SITE]`
* You should see the nudge.
![image](https://user-images.githubusercontent.com/538790/58116470-a57e3e80-7bfc-11e9-9dff-51f4d85b2c04.png)

* Click on "UPGRADE" and you should see the banner on the plans page.
![image](https://user-images.githubusercontent.com/538790/58116444-926b6e80-7bfc-11e9-94cb-0ab2755df3f7.png)

* Now do the same for a Jetpack site and you should see the nudge and banner again, this time the banner should mention the "JPSALE20" coupon code, instead of "WPSAVE20".
